src/app/app.component.html

```diff
@@ -1,2 +1,2 @@
 <h1>NgWig Demo</h1>
-<ng2wig></ng2wig>
+<ng2wig plugins="demo"></ng2wig>
```

Comment:
That was my intention initially. But I gave second thoughts since I read in the Angular official documentation that is used by convention to configure a module's providers. What is your opinion on that? It also seems more reasonable and clear, to me, to use **forRoot**.